pages/linux/apt-get.md

```diff
@@ -15,6 +15,10 @@
 
 `apt-get remove {{package}}`
 
+- Remove a package and configuration files:
```

Comment:
```suggestion
- Remove a package and its configuration files:
```
I think it's clearer this way. What do you think?

Reply:
Yes, sure.